tests/conftest.py

```diff
@@ -54,7 +54,22 @@ def key():
 
 @pytest.fixture
 def document_has_metadata(service_id, document_id, key, rmock, client):
```

Comment:
Regarding the fixtures document_has_metadata and document_has_metadata_verify_email, considering that the distinction between the two is whether "verify_email" is true or false, could we just have one fixture "document_has_metadata" and then pass in the value of "verify_email" (True or false)?

Reply:
It's a good point, but I think in this case I prefer having separate fixtures. I've made the original one more explicit now as `document_has_metadata_no_verification`.

Parameterising test fixtures to take in a `verify_email` arguments makes them a bit more awkward to use and harder to read. I'm broadly OK with a little repetition in texts/fixtures, and as we only have 2 cases it feels premature to optimise here.